.github/workflows/build-docker-image.yaml

```diff
@@ -0,0 +1,76 @@
+name: Build - Docker image
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'simple-data-backend/**'
+    # trigger events for SemVer like tags
+    tags:
+      - '*.*.*'
+      - '*.*.*-*'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'simple-data-backend/**'
+
+env:
+  IMAGE_NAMESPACE: "tractusx"
+  IMAGE_NAME: "simple-data-backend"
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # Create SemVer or ref tags dependent of trigger event
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}
+          # Automatically prepare image tags; See action docs for more examples.
+          # semver patter will generate tags like these for example :1 :1.2 :1.2.3
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}
+            type=semver,pattern={{major}}.{{minor}}
+
```

Comment:
In the case that other base images are chosen, the workflow should be enabled for multi-platform build:
https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/755/files#diff-8af365c7bcc076165459d580c352a85552bd4c63ef83d773473ba80584bd37e2R96
https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/755/files#diff-8af365c7bcc076165459d580c352a85552bd4c63ef83d773473ba80584bd37e2R132